services/wiki/wiki.go

```diff
@@ -43,6 +43,11 @@ func NormalizeWikiName(name string) string {
 	return strings.ReplaceAll(name, "-", " ")
 }
 
+// NameToUnescapedFilename converts a wiki name to its corresponding filename without url query escape.
+func NameToUnescapedFilename(name string) string {
```

Comment:
// Pass in the raw unchanged name
	// Add ".md" suffix if not there.

where is the check if it end with .md ?

Reply:
That's the recommended step from [this inline review](https://github.com/go-gitea/gitea/pull/16139#discussion_r655600166). I notice that the original `NameToFilename()` didn't do that either, so do I need to add that logic to both `NameToUnescapedFilename()` and `NameToFilename()` or just remove that comment?

Reply:
The function then seems bit useles and we should remove it

Reply:
I prefer to remove the comment and added a commit to remove these two lines of comment, let me know if you think I should add the check instead of remove it, thanks!

Reply:
> The function then seems bit useles and we should remove it then

Oh! Sorry I missed this comment. A new commit added to remove that function now.